events/wmt.md

```diff
@@ -0,0 +1,99 @@
+---
+parent: Events
+title: WMT
+description: Conference on Machine Translation
+name: WMT
+---
+
+**WMT** is the main event for machine translation and machine translation research.  The conference is held annually in connection with larger conferences on natural language processing.
+
+> The conference aims to bring together academic scientists, researchers and industry representatives to exchange and share their experiences and research results. WMT plays a key role for the entire industry of computational linguistics and machine translation.
+
+In 2006, the first **Workshop on Machine Translation** was held at the NAACL (North American Chapter of the Association for Computational Linguistics Annual Meeting).
+
+In 2016, with the rise of [neural machine translation](approaches/neural-machine-translation.md), WMT became a conference of its own.  The **Conference on Machine Translation** is still mainly known as *WMT*.
+
+Universities, research laboratories and big technology companies consistently participate in the conference and are represented in the organising committee.
+
+## Shared tasks
+
+WMT hosts shared tasks competitions prior to the conference. During the shared tasks, researchers tackle specific challenges from a list of themes. Researchers then compare their techniques against the techniques of other researchers in the field.
+
+During the main conference, researchers present the results of the shared tasks and winners are announced.
+
+WMT started in 2006 with a *translation* task. In the following editions, WMT included themes on all aspects of machine translation, corpus preparation, training, and evaluation.
+
+
+#### Translation tasks
+
+- General machine translation task (former News task)
```

Comment:
I'd say just add a sentence above.

> The main task is the general machine translation task.  Until 2022, it was known as the news task because traditionally the content to be translated was news articles.